api_tests/tests/sanity.ts

```diff
@@ -6,6 +6,60 @@ import SwapFactory from "../src/actors/swap_factory";
 // ******************************************** //
 
 describe("Sanity", () => {
+    it(
+        "invalid-path-yields-405",
```

Comment:
405 means something entirely different and you are also not asserting what you are saying you are asserting :D

Reply:
I'll give you that I am not necessarily asserting its a 405 since I only check for 'name: Error' but curl tells me that its a 405 being returned. If that's the wrong thing to return that's upstream's business I believe since this is, if I understand correctly, a warp error that we do not control.

Reply:
or are you saying the term 'path' is wrongly used when talking about an endpoint/route? Please do correct me on HTTP stuff, I am not a web developer so I definitely need these things pointed out to me if I get them wrong.